### PR TITLE
Add press hold for send command

### PR DESCRIPTION
**Pull request by ehendrix23**
**Description:**

Documentation with example for the new hold_secs parameter within the send_command.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19650

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/remote.harmony.markdown [hunk 1]
```diff
@@ -75,6 +80,8 @@ Upon startup one file will be written to your Home Assistant configuration direc
 - List of all programmed device names and ID numbers
 - List of all available commands per programmed device
 
+This file will be overwritten whenever the Harmony HUB has a new configuration, there is no need to restart HASS.
```
Comment (DubhAd) [inline source/_components/remote.harmony.markdown]: HASS -> Home Assistant

File: source/_components/remote.harmony.markdown [hunk 2]
```diff
@@ -118,28 +131,94 @@ Send a single command or a set of commands to one device, device ID and availabl
...
+                "VolumeDown",
+                "Mute"
+            ],
+            "id": "428297615"
+        }        
+    }
+}
+```
+
+
```
Comment (DubhAd) [inline source/_components/remote.harmony.markdown]: Stray blank line

File: source/_components/remote.harmony.markdown [hunk 3]
```diff
@@ -118,28 +131,94 @@ Send a single command or a set of commands to one device, device ID and availabl
...
+  hold_secs: 3
+```
+
+### {% linkable_title Service `remote.change_channel` %}
+
+Sends the change channel command to the Harmony HUB 
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on a specific remote, else target all.
```
Comment (DubhAd) [inline source/_components/remote.harmony.markdown]: Not a doc point, but I believe Home Assistant is moving away from that, and requiring that `all` be used as the entity ID to target all devices.

**Review by DubhAd — DISMISSED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
@ehendrix23 There are some review changes requested and there appears to be a merge conflict. Could you please take a look? 👍

**Comment by ehendrix23** [comment]
Updates done :-)

**Comment by frenck** [comment]
Thanks, @ehendrix23 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**